public/index.html

```diff
@@ -6,8 +6,9 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
 
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ecomplus/storefront-twbs@3/dist/storefront-twbs.min.css">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/ant-design-vue@1/dist/antd.min.css">
+    <link rel="stylesheet" href="https://app.e-com.plus/admin.2b7bd29a11cf34c6471c.css">
+    
+    <!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/ant-design-vue@1/dist/antd.min.css"> -->
```

Comment:
```suggestion
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/ant-design-vue@1/dist/antd.min.css">
```

Como eu comentei com você na call, só podemos tirar esse cara aqui quando realmente tivermos finalizado a refatoração, por enquanto ele ainda aplica estilos a vários componentes.